string class_name
float32 confidence
float32 gripper_rotate
geometry_msgs/Pose target_pose